### PR TITLE
Update SherlockMLAuth docstring.

### PR DESCRIPTION
**Pull request by sbalian**
Just added `services` in the two URLs.

### CODE REVIEW

File: sherlockml/clients/auth.py [hunk 1]
```diff
@@ -64,14 +64,15 @@ class SherlockMLAuth(AuthBase):
...
-    >>> auth = SherlockMLAuth('https://hudson.example.sherlockml.net',
+    >>> auth = SherlockMLAuth('https://hudson.services.example.sherlockml.net',
                               your_client_id, your_client_secret)
 
     then pass it as the ``auth`` argument when making a request with
     ``requests``:
 
     >>> import requests
-    >>> requests.get('http://service.example.sherlockml.net', auth=auth)
+    >>> requests.get('https://service.services.example.sherlockml.net',
```
Comment (janfreyberg) [inline sherlockml/clients/auth.py]: can we replace this with `https://servicename.services.example.sherlockml.net` to make it less ambiguous?

Comment (acroz) [reply sherlockml/clients/auth.py]: :+1:

Comment (sbalian) [reply sherlockml/clients/auth.py]: Changed.

**Comment by janfreyberg** [comment]
Nice! Good to merge.

**Comment by sbalian** [comment]
Thanks @janfreyberg . Can you merge it? (I'm not authorised for some reason)

**Comment by janfreyberg** [comment]
Me neither! I think only @acroz is

**Pull request merged**